src/Compilers/VisualBasic/Portable/LanguageVersion.vb

```diff
@@ -21,6 +21,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         VisualBasic15_5 = 1505
         VisualBasic16 = 1600
         VisualBasic16_9 = 1609
+        VisualBasic17_13 = 1713
```

Comment:
Please add the new language version and feature in:
- [https://github.com/dotnet/vblang/blob/main/Language-Version-History.md](https://github.com/dotnet/vblang/blob/main/Language-Version-History.md)
- [https://github.com/dotnet/roslyn/blob/main/docs/Language%20Feature%20Status.md](https://github.com/dotnet/roslyn/blob/main/docs/Language%20Feature%20Status.md)

Reply:
Is there an explanation as to what this enables VB to do?